src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -161,6 +185,7 @@ const NavSearchBar = () => {
       <InputField
         value={searchInputValue}
         inputClassName="NavSearchBar__search-input"
+        ref={inputRef}
```

Comment:
[nit] Prefer `ref` to be the first prop:

```suggestion
      <InputField
        ref={inputRef}
        value={searchInputValue}
        inputClassName="NavSearchBar__search-input"
```

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/1416/commits/2dc066902d7fe56f50eb54583cc33294a7f246db